openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java

```diff
@@ -186,7 +186,8 @@ public void setUp(StfCoreExtension test, StfSharedClassesExtension sharedClasses
 		// See whether the test data (20000 classes and jars) already exist.
 		// If not, create them in the first -systemtest-prereqs directory (or the default location if no -systemtest-prereqs were supplied).
 		DirectoryRef sharedClassesDataDir = null;   // This will be initialised when we find it.
-		String dataSubdir = "sharedClassesTestData/v1";
+		int javaVersion = test.env().primaryJvm().getJavaVersion();
```

Comment:
Not sure if `test.env()` works in all situations, a same case to get javaVersion is using `environmentCore` [link](https://github.com/eclipse-aqavit/openj9-systemtest/blob/0cc1cdb8e33b41de4478f8f07d53a461905e5f49/openj9.stf.extensions/src/stf.extensions/net/openj9/stf/sharedClasses/StfSharedClassesExtension.java#L301)

Reply:
The `test.env()` was also used in other places, and it passed in all grinder links mentioned in https://github.com/eclipse-aqavit/openj9-systemtest/pull/156#issuecomment-2165777969. So probably will keep it then.